_rules/html-page-lang-matches-default-ucwvc8.md

```diff
@@ -0,0 +1,280 @@
+    forConformance: false
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: satisfied
+input_aspects:
+  - DOM tree
+  - Accessibility tree
+  - CSS Styling
+  - Language
+acknowledgments:
+  authors:
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any [document element][] if it is an `html` element for which all of the following are true:
+
+- The [document element][] has a `lang` attribute with a value that is a [valid language tag][]; and
+- The [document element][] is in a [top-level browsing context][]; and
+- The [document element][] has a [content type][] of `text/html`; and
+- The [document element][] has a defined [default page language][].
+
+## Expectation
+
+For each test target, the [primary language][] of the [valid language tag][] matches the [default page language][] of the test target.
+
+## Assumptions
+
+- This rule assumes that the default human language of a page, as described in WCAG 2, can be determined by counting the number of words used in each language. If the default language needs to be derived in some other way (such as frequency analysis, mutual information based distance, …), this rule may fail while [Success Criterion 3.1.1: Language of Page](https://www.w3.org/TR/WCAG21/#language-of-page) is still satisfied.
+
+- The language of the page can be set by other methods than the `lang` attribute, for example using HTTP headers or the `meta` element. These methods are not supported by all assistive technologies. This rule assumes that these other methods are insufficient to satisfying [Success Criterion 3.1.1: Language of Page](https://www.w3.org/TR/WCAG21/#language-of-page).
+
+- This rule assumes that user agents and assistive technologies can programmatically determine [valid language tags](#valid-language-tag) even if these do not conform to the [BCP 47][] syntax.
+
+- This rule assumes that [grandfathered tags][] are not used as these will not be recognized as [valid language tags](#valid-language-tag).
+
+- This rule assumes that `iframe` title elements are not exposed to assistive technologies and so does not consider them as part of the [default page language][].
+
+## Accessibility Support
+
+_There are no major accessibility support issues known for this rule._
+
+## Background
+
+- [Understanding Success Criterion 3.1.1: Language of Page](https://www.w3.org/WAI/WCAG21/Understanding/language-of-page.html)
+- [H57: Using language attributes on the html element](https://www.w3.org/WAI/WCAG21/Techniques/html/H57)
+- [BCP 47: Tags for Identifying Languages](https://www.ietf.org/rfc/bcp/bcp47.txt)
+- [The `lang` and `xml:lang` attributes](https://html.spec.whatwg.org/multipage/dom.html#the-lang-and-xml:lang-attributes)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This page has a [default language][default page language] of English, which matches the language indicated in the `lang` attribute.
```

Comment:
We are not explaining why this page has a default language of English. Adding "as all of its words are English" or something along these lines may help clarity and consistency.

Reply:
This description is also a bit another way round with the applicability and expectation of this rule. I guess the description of examples should be according to the [test case design](https://github.com/act-rules/act-rules.github.io/blob/develop/pages/design/rule-design.md#test-cases) that is the example has or does not have or contains the necessary condition to be met and expectation. Indirectly, conditions in the case of a test rule is "Applicability" and then comes "Expectation". 
It does not mean that this description is not sufficient or incorrect but it should be according to the template we defined in [test case design](https://github.com/act-rules/act-rules.github.io/blob/develop/pages/design/rule-design.md#test-cases).
According to the description of the rule plus the applicability and expectation, this rule checks that the primary language subtag of the page language matches the default language of the page. It is similarly the same if we say the default language of this page matches the primary language subtag of the valid language tag of the page but another way round against the description of the rule.
```suggestion
This page has a [primary language][] English ("en") set in the [valid language tag][] which matches the [default page language][] that is English.
```